components/function-controller/internal/controllers/kubernetes/shared.go

```diff
@@ -19,6 +22,8 @@ const (
 type Config struct {
 	BaseNamespace                 string        `envconfig:"default=kyma-system"`
 	ExcludedNamespaces            []string      `envconfig:"default=istio-system;knative-eventing;kube-node-lease;kube-public;kube-system;kyma-installer;kyma-integration;kyma-system;natss;compass-system"`
+	RoleRequeueDuration           time.Duration `envconfig:"default=1m"`
+	RoleBindingRequeueDuration    time.Duration `envconfig:"default=1m"`
```

Comment:
add those values to helm chart as envs, so that we can override them during installation if that's needed

Reply:
done